docs/usage/configuration-options.md

```diff
@@ -2939,9 +2841,8 @@ The above will configure `rangeStrategy` to `pin` only for the npm package `angu
 {
   "packageRules": [
     {
-      "matchDatasources": ["npm"],
-      "matchPackageNames": ["@angular/*", "!@angular/abc"],
-      "groupName": "Angular"
+      "matchPackagePatterns": ["^angular"],
+      "rangeStrategy": "replace"
```

Comment:
```suggestion
      "matchDatasources": ["npm"],
      "matchPackageNames": ["@angular/*", "!@angular/abc"],
      "groupName": "Angular"
```

> The above will group together any npm package which starts with `@angular/` except `@angular/abc`.

https://github.com/renovatebot/renovate/pull/28602/files#diff-be0a39f342380493f17c380bbd694f1f05d18bdd8d7cecf174369084952102deR2851
This does not fit the description

Reply:
I thought it does, how come you think why not?

Reply:
The description is below not above the code snippet. Notice the `The above will` prefixes.